content/docs/command-reference/run.md

```diff
@@ -144,6 +144,11 @@ $ dvc run -n my_stage "./my_script.sh > /dev/null 2>&1"
 $ dvc run -n my_stage './my_script.sh $MYENVVAR'
 ```
 
+Using `dvc run` with the `--no-exec` option doesn't execute the `command` and
+can be used to
+[add output to a stage](/docs/user-guide/how-to/add-output-to-stage) without
+re-running it.
+
```

Comment:
- This doesn't belong in the **The `command` argument** section, please find another place to insert this.
- It shouldn't focus on the `command`, probably not even mention it. It's about adding a missing output after the stage has already been run.